lib/rules/one-suite-per-file.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+/**
+ * @fileoverview Disallow multiple top-level suites in a single file
+ * @author Alexander Afanasyev
+ */
+
+var defaultSuiteNames = [ 'describe', 'context', 'suite' ];
+
+module.exports = function (context) {
+    var stack = [],
+        topLevelDescribes = [],
+        suiteNames = context.options[0] ? context.options[0] : defaultSuiteNames;
+
+    return {
+        CallExpression: function (node) {
+            var callee = node.callee;
+            if (callee && callee.name && suiteNames.indexOf(callee.name) > -1) {
```

Comment:
Should we also support modifiers as e.g. `descirbe.only`, `describe.skip` etc.?

You could also reuse the [`isDescribe`](https://github.com/lo1tuma/eslint-plugin-mocha/blob/4b7963346efed34a46dc51fa2ad7958b8f3cb2e4/lib/util/ast.js#L13) function.